lib/src/Base/Stat/CorrelationAnalysis.cxx

```diff
@@ -26,105 +26,163 @@
 
 BEGIN_NAMESPACE_OPENTURNS
 
-/* Compute the Pearson correlation coefficient between the component number index of the input sample and the 1D outputSample */
+/**
```

Comment:
Sorry for the minor remark...
Following what @JPelamatti proposed, I was more fond of the names "firstSample / secondSample" as these metrics are "two-sample statistics". Using "sampleX / sampleY" still denotes (implicitly) an input-output relationship which might be confusing. Again, this is a tiny detail.

Reply:
Done. Also removed further references to input/output variables in the `CorrelationAnalysis` API doc.